<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationFrame</class>
 <widget class="QMainWindow" name="ApplicationFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HeliView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabPane">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPaneCamera">
       <attribute name="icon">
        <iconset resource="HeliView.qrc">
         <normaloff>:/data/camera.png</normaloff>:/data/camera.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QHBoxLayout" name="tabPaneCameraLayout"/>
      </widget>
      <widget class="QWidget" name="tabPaneSensors">
       <attribute name="icon">
        <iconset resource="HeliView.qrc">
         <normaloff>:/data/graph.png</normaloff>:/data/graph.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Sensors</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabPaneSensorsLayout">
        <item>
         <widget class="QLabel" name="lblNoAxes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>No Axes Seleced</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="axisControlLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupXAxis">
            <property name="title">
             <string>X Axis</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QCheckBox" name="cbShowXF">
               <property name="text">
                <string>Acceleration</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="cbShowXUF">
               <property name="text">
                <string>Raw Velocity</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Filtered</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="checkBox_4">
               <property name="text">
                <string>Paused</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupYAxis">
            <property name="title">
             <string>Y Axis</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QCheckBox" name="cbShowYF">
               <property name="text">
                <string>Acceleration</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="cbShowYUF">
               <property name="text">
                <string>Raw Velocity</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="checkBox_2">
               <property name="text">
                <string>Filtered</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="checkBox_5">
               <property name="text">
                <string>Paused</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupZAxis">
            <property name="title">
             <string>Z Axis</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QCheckBox" name="cbShowZF">
               <property name="text">
                <string>Acceleration</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="cbShowZUF">
               <property name="text">
                <string>Raw Velocity</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="checkBox_3">
               <property name="text">
                <string>Filtered</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="checkBox_6">
               <property name="text">
                <string>Paused</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPaneVirtual">
       <attribute name="icon">
        <iconset resource="HeliView.qrc">
         <normaloff>:/data/virtual.png</normaloff>:/data/virtual.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Virtual</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabPaneVirtualLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>Control Overrides</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="btnTakeoff">
       <property name="text">
        <string>Takeoff</string>
       </property>
       <property name="icon">
        <iconset resource="HeliView.qrc">
         <normaloff>:/data/go-up.png</normaloff>:/data/go-up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLanding">
       <property name="text">
        <string>Landing</string>
       </property>
       <property name="icon">
        <iconset resource="HeliView.qrc">
         <normaloff>:/data/go-bottom.png</normaloff>:/data/go-bottom.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOverride">
       <property name="text">
        <string>Manual Override</string>
       </property>
       <property name="icon">
        <iconset resource="HeliView.qrc">
         <normaloff>:/data/manual.png</normaloff>:/data/manual.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKillswitch">
       <property name="text">
        <string>Killswitch</string>
       </property>
       <property name="icon">
        <iconset resource="HeliView.qrc">
         <normaloff>:/data/kill.png</normaloff>:/data/kill.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>Status Indicators</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Connection:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vehicle:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tracking:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Elevation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Battery:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="windowTitle">
    <string>Command Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QTextEdit" name="txtCommandLog">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Liberation Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Connecting.... OK&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Target found, seeking...&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;In range, hovering...&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Enter manual mode...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="HeliView.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbShowXF</sender>
   <signal>toggled(bool)</signal>
   <receiver>ApplicationFrame</receiver>
   <slot>onShowXFChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowXUF</sender>
   <signal>toggled(bool)</signal>
   <receiver>ApplicationFrame</receiver>
   <slot>onShowXUFChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowYUF</sender>
   <signal>toggled(bool)</signal>
   <receiver>ApplicationFrame</receiver>
   <slot>onShowYUFChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowYF</sender>
   <signal>toggled(bool)</signal>
   <receiver>ApplicationFrame</receiver>
   <slot>onShowYFChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowZF</sender>
   <signal>toggled(bool)</signal>
   <receiver>ApplicationFrame</receiver>
   <slot>onShowZFChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowZUF</sender>
   <signal>toggled(bool)</signal>
   <receiver>ApplicationFrame</receiver>
   <slot>onShowZUFChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabPane</sender>
   <signal>currentChanged(int)</signal>
   <receiver>ApplicationFrame</receiver>
   <slot>onTabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShowXFChanged(bool)</slot>
  <slot>onShowXUFChanged(bool)</slot>
  <slot>onShowYUFChanged(bool)</slot>
  <slot>onShowYFChanged(bool)</slot>
  <slot>onShowZFChanged(bool)</slot>
  <slot>onShowZUFChanged(bool)</slot>
  <slot>onTabChanged(int)</slot>
 </slots>
</ui>
